### ISSUE TITLE
Extracting and Reusing parts of response - Does not seem to work with formData

### ISSUE THREAD

**Issue by sithara** [shoreditch-ops/artillery#491]
I'm trying to use the response received from `/requests/create` to be fed into `/attachments/upload`. But while printing the request , I can see that "{{pid}}" is being considered as a string and not a response value.

Please find the code snippet below:

```
config:
  target: "http://127.0.0.1:8000"
  plugins:
    http-file-uploads: {}
  http:
    timeout: 10
  phases:
    - duration: 2
      arrivalRate: 1
scenarios:
  - flow:
      - post:
          url: "/requests/create"
          capture:
            json: $.attachment.attachment_id
            as: "pid"
      - post:
          url: "/attachments/upload"
          formData:
            attachment_id: "{{pid}}"
            attachment_file:
              fromFile: "myFile.pdf"

```
We are using artillery-pro for the file upload. This works correctly with `form` but not with `formData` - I cannot use `form` since I also need to upload the file.

Appreciate any help!

**Comment by hassy** [comment]
@sithara if you capture the field you need explicitly, it should work as expected. Something like:

```yaml
scenarios:
  - flow:
      - post:
          url: "/requests/create"
          #
          # capture the value of $.status into a variable "status"
          #
          capture:
            - json: "$.status"
              as: status
      - post:
          url: "/attachments/upload"
          formData:
            attachment_id: "{{status}}"
            attachment_file:
               fromFile: "file.pdf"
```

(The documentation mentions the catch-everything `$` variable as I've just realized, but that's an old feature which has been removed. Apologies for the misleading documentation, it's been updated now.)

**Comment by sithara** [comment]
@hassy Thank you for the quick response. But the issue is I cannot get `{{status}}` within `formData`. It works fine outside `formData`

**Comment by hassy** [comment]
Oh I see, thanks! That looks like a bug, but we expect the turnaround on fixing & shipping a new version to be very quick. I'll keep you posted. Thanks @sithara!

**Comment by sithara** [comment]
Thanks much @hassy  .  Expecting the issue to be fixed soon :)

**Comment by hassy** [comment]
Fixed in #492 and available in Artillery 1.6.0-20.
